socorro/unittest/external/http/test_correlations.py

```diff
@@ -25,9 +27,20 @@
 
 
 class Response(object):
-    def __init__(self, content=None, status_code=200):
-        self.content = content
+    def __init__(self, content=None, status_code=200, gzip=False):
```

Comment:
``gzip`` is a terrible name for a boolean. Especially since it "conflicts" with the gzip lib that you import. Could you consider something like ``do_gzip`` instead?